flink/v1.13/flink/src/test/java/org/apache/iceberg/flink/TestFlinkUpsert.java

```diff
@@ -164,38 +169,31 @@ public void testUpsertAndQuery() {
   public void testPrimaryKeyEqualToPartitionKey() {
     // This is an SQL based reproduction of TestFlinkIcebergSinkV2#testUpsertOnDataKey
```

Comment:
Can you remove this comment as well while you're updating this? It's no longer applicable really and not hepful.

Thank you!